django/db/backends/base/base.py

```diff
@@ -71,6 +71,15 @@ def __init__(self, settings_dict, alias=DEFAULT_DB_ALIAS,
         self.allow_thread_sharing = allow_thread_sharing
         self._thread_ident = thread.get_ident()
 
+        # a list of no-argument functions to run when the transaction commits;
+        # each entry is an (sids, func) tuple, where sids is a list of the
```

Comment:
if the `sids` element in this tuple is changed from a list to a set, this comment would also need to be updated accordingly